cmd/root.go

```diff
@@ -59,6 +59,7 @@ func Init(version, artifactArch string) {
 	addCommand(rootCmd, validateCmd)
 	addCommand(rootCmd, openapiCmd)
 	addCommand(rootCmd, migrateCmd)
+	addCommand(rootCmd, AskCmd)
```

Comment:
If we want this to show up in interactive mode, we need to probably hide another command. Right now we have the max we can fit into a single page. Alternatively we can hide this one from interactive mode

Reply:
Either way is good with me, tbh. Happy to hide it from here if our others are more valuable to have, and they might be.